storage/store/execution_fork_evidence.go

```diff
@@ -0,0 +1,60 @@
+package store
```

Comment:
Do we still need the corresponding badger modules, can we delete them, so that we can see a better diff of the changes.

Reply:
> Do we still need the corresponding badger modules, can we delete them, so that we can see a better diff of the changes.

@zhangchiqing There are no corresponding badger moduless for store.

Store functionality (store and retrieve) is implemented in `module/mempool/consensus/exec_fork_suppressor.go`.

These removed Badger functions are in that file:
- `checkExecutionForkEvidence()`
- `storeExecutionForkEvidence()`

Reply:
You are right. Thanks.